frontend/src/locales/zh-CN.yml

```diff
@@ -945,12 +946,13 @@ deployment-config:
     at-least-one-stage: 至少需要 1 个阶段
     stage-name-required: 需要指定阶段名称
     at-least-one-selector: 每个阶段至少需要 1 个选择条件
-    env-in-selector-required: 每个阶段都需要有 "environment In" 选择条件
-    env-selector-must-has-one-value: '"environment In" 必须有且只有 1 个值'
+    env-in-selector-required: 每个阶段都需要有 "Environment In" 选择条件
```

Comment:
Maybe just say "Environment" since that is good enough? Otherwise we might need to translate "In" ?

Reply:
I think we might need to translate "In" since "Environment Exists" is not an acceptable selector.
So we may translate "In" to "包含" and "Exists" to "存在", wdyt?

Reply:
At the code level we always use "in", so we either omit it or keep it as is.

Reply:
I prefer keeping it and using lowercase instead.
An awkward thing is that we use "In" and "Exists" in the JSON API. So I just kept it as-is in the last version.
Anyway, I think "Environment in" is better than "Environment In" when reading. So I'm gonna change it to lowercase when displaying.